distribution/changelog.txt

```diff
@@ -1,5 +1,6 @@
 0.2.5+ (in development)
 ------------------------------------------------------------------------
+- Feature: [#11306] Replacing path textures do not remove additions on it. It handles the switch from queue line to standard path and vice versa.
```

Comment:
This needs to be moved to the 0.2.6+ section. But in order to do so, you must first rebase this pull request.